draft-ietf-rtcweb-jsep.xml

```diff
@@ -1765,29 +1765,34 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
-                    pending remote description according to the rules
-                    defined for Trickle ICE.
-                    If the MID, m-line index, or candidate string provided in the ICE candidate is
-                    invalid, an error is generated.
-                    If present, ufrag is used to determine which
-                    generation of ICE candidates the candidate belongs to;
-                    this resolves ambiguities when trickle ICE is in progress
-                    during an ICE restart. If the ufrag is present and
-                    does not match any remote description, an error is generated.
-                    If the ufrag is absent, the candidate MUST be assumed to belong to
-                    the most recently applied remote description.
-                    Connectivity checks will
-                    be sent to the new candidate.
-                    </t>
+                  <t>
+                    The addIceCandidate method provides a remote
+                    candidate to the ICE Agent, which, if parsed successfully, will be
+                    added to the current and/or pending remote description according to
+                    the rules defined for Trickle ICE.
+                    The pair of MID and ufrag is used to
+                    determine the m= section and ICE candidate generation to which the
+                    indication belongs. If the MID is not present, the m= line index is used
+                    to look up the locally generated MID (see <xref
+                    target="sec.applying-a-remote-desc"/>), which is used in place of
+                    the MID. If these values or the candidate string are
+                    invalid, an error is generated. The purpose of
+                    the ufrag is to resolve ambiguities when trickle ICE is in progress
+                    during an ICE restart.  If the ufrag is absent, the candidate MUST be
+                    assumed to belong to the most recently applied remote description.
+                    Connectivity checks will be sent to the new candidate.
+                  </t>
 
-                    <t>This method can also be used to provide an
-                    end-of-candidates indication (as defined in <xref
-                    target="I-D.ietf-ice-trickle" />) to the ICE Agent
-                    for all media descriptions in the last remote
-                    description.</t>
+                  <t>
+                    This method can also be used to provide an
+                    end-of-candidates indication to the ICE Agent,
+                    as defined in <xref target="I-D.ietf-ice-trickle" />).
+                    The MID and ufrag are used as described above to determine
+                    the m= section and ICE generation for which candidate
+                    gathering is complete. If both the MID and m= line index
```

Comment:
That's a good point. I think we do need it because the text above would prohibit this case.